test/base/src/commonTest/kotlin/io/realm/test/NotificationTests.kt

```diff
@@ -39,23 +39,27 @@ interface NotificationTests {
     fun closingRealmDoesNotCancelFlows()
 
     @Test
-    fun addChangeListener() {
+    @Ignore
```

Comment:
Doesn't ignoring the test goes completely against the concept of having the test in the interface. I think the issue is that we didn't get to the end designing this. There should probably never be an empty default implementation here and if none of the implementation classes actually implements it then it should probably just not be defined. At least not without an issue reference stating what is missing. 

There are also other issues, that implementations mess up the JVM test, so maybe we should just leave it how you have fixed it and then do a separate issue to revisit the pattern.

Reply:
test re-enabled